.github/workflows/docker.yml

```diff
@@ -6,7 +6,6 @@
 name: "Docker"
 on:
   push:
-    branches: [ "hemi" ]
```

Comment:
Why this change?

Reply:
ah this is just to be able to push a docker image to use for testing in repos that require the docker image itself (`docker-compose_mainnet.yml` in `heminetwork` requires a docker image).

I can revert it, and I think that @joshuasing would request that I do 😄 , but I still like the ability to push an image per commit for testing, rather than it having to be on a certain branch

Reply:
okay, then I leave this to you and @joshuasing